tests/test_django_log_formatter.py

```diff
@@ -223,6 +224,23 @@ def test_logs_log_personally_identifiable_information_when_log_sensitive_user_da
         assert TEST_FIRST_NAME in raw_log
         assert TEST_LAST_NAME in raw_log
 
+    @patch("ddtrace.trace.tracer.current_span")
+    def test_logs_log_datadog_required_values(self, mock_ddtrace_span, caplog):
```

Comment:
I think we should probably have a test that they are empty when the environment variables are not set and only set the environment variables for this test.

Reply:
That sounds fair, I'll get a new test written up for the DataDog trace dict values.